Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
My general position is that we maintain the status quo of not rewriting the `npm` handling unless something breaks, because every time we touch this section significantly it becomes a major, major issue 😅. 

> for a working npm@5.2.0 update exposing npx

I did mention [upstream](https://github.com/npm/npm/pull/17685#issuecomment-313946044) if we wanted to include that in the `node` formula we'd need to add a few more lines to `postinstall`, so, hopefully someone remembers 😄.